### ISSUE TITLE
Section indentation not triggering error

### ISSUE THREAD

**Issue by kannes** [terrencepreilly/darglint#10]
"Sections should be indented two spaces, except for the heading." says https://google.github.io/styleguide/pyguide.html#383-functions-and-methods (and I wonder why it says two while their example uses four...)

Consider 

```
def f(foo):
    """foobar foobars foobar

         Args:
             foo: foobar
            
           Returns:
              foobarred bar
    """
    return "bar"
```

I would have expected a error about the messed up indentation. But got a successful linting run instead:

```
$ darglint -v 2 f.py 
$ 
```

I am on 0.5.1.

**Comment by kannes** [comment]
Ah, this would be another stylistic error.

I just like reporting whatever my students end up writing ;)

**Comment by terrencepreilly** [comment]
So, currently darglint expects indentation to be four spaces (as all the examples show.)  I actually want to change this to be any indentation amount, so long as it is consistent.  I'll probably add this at the same time I handle the other style errors (since this will also require updates to the parser.)

**Comment by terrencepreilly** [comment]
Probably, though, an issue/merge request should be put into the Google style guide.

**Comment by terrencepreilly** [comment]
Okay, I've submitted an issue [here](https://github.com/google/styleguide/issues/417).
